service/project.go

```diff
@@ -248,7 +243,57 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	var hook *model.GithubHook
+	hook, err = model.FindGithubHook(responseRef.Owner, responseRef.Repo)
+	if err != nil {
+		uis.LoggedError(w, r, http.StatusInternalServerError, err)
+		return
+	}
+	origGithubWebhookEnabled := (hook != nil)
+	if hook == nil {
+		var hookID int
+		hookID, err = uis.setupGithubHook(responseRef.Owner, responseRef.Repo)
+		if err != nil {
+			grip.Error(message.WrapError(err, message.Fields{
+				"source":  "project edit",
+				"message": "can't setup webhook",
+				"project": id,
+				"owner":   responseRef.Owner,
+				"repo":    responseRef.Repo,
+			}))
+			// don't kill it here, sometimes people change
```

Comment:
I think the intention was not to stop saving the project just because we can't set up a hook. It could be the user is changing the repo to track a private repo we don't have access to (outside the 10gen, mongo,... orgs).

Reply:
I clarified the comment.